tests/integration/suite/test_app.py

```diff
@@ -86,6 +87,69 @@ def test_app_wordpress(admin_pc, admin_mc):
     wait_for_workload(client, ns.name, count=2)
 
 
+@pytest.mark.skip(reason="istio disabled")
```

Comment:
A disabled test does not have much use. If this cannot be legitimately tested as an integration test, then remove it.